### ISSUE TITLE
New version plotly error

### ISSUE THREAD

**Issue by remiolsen** [ewels/MegaQC#9]
Pip will install version `3.0_rc1` of plotly. This doesn't work (never mind my python 3.6 experimentation):

```python
megaqc initdb
This is MegaQC v0.1.dev0

Traceback (most recent call last):
  File "/Users/remi-andreolsen/miniconda3/bin/megaqc", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/Users/remi-andreolsen/code/MegaQC/scripts/megaqc", line 31, in <module>
    cli()
  File "/Users/remi-andreolsen/miniconda3/lib/python3.6/site-packages/click-6.7-py3.6.egg/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/remi-andreolsen/miniconda3/lib/python3.6/site-packages/Flask-0.12-py3.6.egg/flask/cli.py", line 374, in main
    return AppGroup.main(self, *args, **kwargs)
  File "/Users/remi-andreolsen/miniconda3/lib/python3.6/site-packages/click-6.7-py3.6.egg/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/remi-andreolsen/miniconda3/lib/python3.6/site-packages/click-6.7-py3.6.egg/click/core.py", line 1061, in invoke
    cmd_name, cmd, args = self.resolve_command(ctx, args)
  File "/Users/remi-andreolsen/miniconda3/lib/python3.6/site-packages/click-6.7-py3.6.egg/click/core.py", line 1100, in resolve_command
    cmd = self.get_command(ctx, cmd_name)
  File "/Users/remi-andreolsen/miniconda3/lib/python3.6/site-packages/Flask-0.12-py3.6.egg/flask/cli.py", line 345, in get_command
    rv = info.load_app().cli.get_command(ctx, name)
  File "/Users/remi-andreolsen/miniconda3/lib/python3.6/site-packages/Flask-0.12-py3.6.egg/flask/cli.py", line 223, in load_app
    rv = self.create_app(self)
  File "/Users/remi-andreolsen/code/MegaQC/scripts/megaqc", line 11, in create_megaqc_app
    from megaqc.app import create_app
  File "/Users/remi-andreolsen/code/MegaQC/megaqc/app.py", line 7, in <module>
    from megaqc.scheduler import init_scheduler
  File "/Users/remi-andreolsen/code/MegaQC/megaqc/scheduler.py", line 6, in <module>
    from megaqc.api.utils import handle_report_data
  File "/Users/remi-andreolsen/code/MegaQC/megaqc/api/__init__.py", line 3, in <module>
    from . import views  # noqa
  File "/Users/remi-andreolsen/code/MegaQC/megaqc/api/views.py", line 8, in <module>
    from megaqc.api.utils import handle_report_data, generate_report_plot, generate_distribution_plot, \
  File "/Users/remi-andreolsen/code/MegaQC/megaqc/api/utils.py", line 16, in <module>
    import plotly.offline as py
  File "/Users/remi-andreolsen/miniconda3/lib/python3.6/site-packages/plotly-3.0.0rc1-py3.6.egg/plotly/__init__.py", line 31, in <module>
    from plotly import (plotly, dashboard_objs, graph_objs, grid_objs, tools,
  File "/Users/remi-andreolsen/miniconda3/lib/python3.6/site-packages/plotly-3.0.0rc1-py3.6.egg/plotly/graph_objs/__init__.py", line 14, in <module>
    from plotly.graph_objs.graph_objs import *  # this is protected with __all__
  File "/Users/remi-andreolsen/miniconda3/lib/python3.6/site-packages/plotly-3.0.0rc1-py3.6.egg/plotly/graph_objs/graph_objs.py", line 802, in <module>
    from plotly.datatypes import FigureWidget
ModuleNotFoundError: No module named 'plotly.datatypes'
```

**Comment by ewels** [comment]
We should pin versions really 👍

**Comment by tsnowlan** [comment]
PR submitted for this

**Comment by ewels** [comment]
Fixed in #11
